AdyenComponents/BACS Direct Debit/Scenes/Input/BACSInputPresenter.swift

```diff
@@ -32,35 +36,49 @@ internal class BACSDirectDebitPresenter: BACSDirectDebitPresenterProtocol {
 
     // MARK: - Initializers
 
-    internal init(view: BACSDirectDebitInputFormViewProtocol,
+    internal init(view: BACSInputFormViewProtocol,
                   router: BACSDirectDebitRouterProtocol,
-                  itemsFactory: BACSDirectDebitItemsFactoryProtocol) {
+                  tracker: BACSDirectDebitComponentTrackerProtocol,
+                  itemsFactory: BACSItemsFactoryProtocol,
+                  amount: Amount?) {
         self.view = view
         self.router = router
+        self.tracker = tracker
         self.itemsFactory = itemsFactory
-        setupItems()
-        setupView()
+        self.amount = amount
     }
 
-    // MARK: - BACSDirectDebitPresenterProtocol
+    // MARK: - BACSDirectDebitInputPresenterProtocol
```

Comment:
there is no "BACSDirectDebitInputPresenterProtocol", right ?

Reply:
yeah, renaming doesn't rename MARKs xD